### PR TITLE
Update Vietnamese Translation P6

### PR DESCRIPTION
**Pull request by Nad-SK**

### CODE REVIEW

**Comment by Nad-SK** [comment]
PASSED!!!! Thanks @ihhub

**Comment by Nad-SK** [comment]
> Hi, @Nad-SK. Please reduce the amount of changes by 28 lines. I also left two other comments. Please have a look.

Hi, @zenseii, how to reduce the changes line??? Open .po file and delete the translate strings???

File: files/lang/vi.po [hunk 1]
```diff
@@ -9875,7 +10032,4 @@ msgid ""
 msgstr ""
 "Kiểm tra phiên bản mới nhất của trò chơi tại\n"
 "https://github.com/ihhub/\n"
-"fheroes2/releases"
-
-#~ msgid "Undo your last action. Press again to redo the action."
-#~ msgstr "Hoàn tác thao tác cuối cùng. Nhấn lại để thực hiện lại thao tác."
+"fheroes2/releases"
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po]: Done

File: files/lang/vi.po [hunk 2]
```diff
@@ -3796,8 +3825,8 @@ msgstr "Phóng đại"
 msgid "Undo"
 msgstr "Hoàn tác"
 
-msgid "Undo your last action."
-msgstr ""
+msgid "Undo your last action. Press again to redo the action."
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po]: Done

**Comment by Nad-SK** [comment]
Hi, @zenseii. It's OK??? Please take a look.

**Comment by ihhub** [comment]
Hi @Nad-SK , just a small suggestion not related to this pull request: please avoid using an extensive amount of punctuation such as ??? because some people might think this as an aggressive manner of request :) A single question mark is sufficient.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @Nad-SK , I left few comments here. Would you mind to take a look at them?

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Nad-SK , thank you so much for the update!

**Pull request merged**